mono/utils/mono-machine.h

```diff
@@ -38,9 +38,9 @@ typedef gsize host_umgreg_t;
 // When dealing with register context, use host_mgreg_t.
 // Or ifndef MONO_CROSS_COMPILE out runtime code.
 #if SIZEOF_REGISTER == 4
-typedef gint32 mgreg_t;
```

Comment:
This block can be removed.

Reply:
Yeah, I wanted to preserve some part of the comment but was taking too long to rewrite it.
I had this on monojenkins squash so it went in asis (good or bad etiquette?).